src/debug/ee/functioninfo.cpp

```diff
@@ -1558,30 +1560,77 @@ DebuggerJitInfo *DebuggerMethodInfo::FindOrCreateInitAndAddJitInfo(MethodDesc* f
 
     _ASSERTE(fd != NULL);
 
-    // This will grab the latest EnC version.
-    TADDR addr = (TADDR) g_pEEInterface->GetFunctionAddress(fd);
+    // The debugger doesn't track Lightweight-codegen methods b/c they have no metadata.
+    if (fd->IsDynamicMethod())
+    {
+        return NULL;
+    }
 
-    if (addr == NULL)
+#ifdef FEATURE_CODE_VERSIONING
```

Comment:
The 'latest' terminology is used by EnC which predates (and doesn't use) FEATURE_CODE_VERSIONING. For code versioning I've attempted to avoid any notion of latest.